src/core/Mixer.cpp

```diff
@@ -868,6 +869,19 @@ MidiClient * Mixer::tryMidiClients()
 	}
 #endif
 
+#ifdef LMMS_HAVE_JACK
+	if( client_name == MidiJack::name() || client_name == "" )
+	{
+		MidiJack * moss = new MidiJack;
```

Comment:
This is nitpicking, but `moss` -> `mjack` because this is not the OSS midi client.

On the other hand, maybe the code for testing all the different backends should just use a generic name for these temporary variables anyway.